ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -310,6 +312,261 @@ describe('a4a_config', () => {
       }
     });
   });
+  it('should serve Delayed Fetch to unlaunched DoubleClick filler',
+    () => {
+      toggleExperiment(win, 'a4aFastFetchDoubleclickLaunched', false, true);
+      element.setAttribute('type', 'doubleclick');
+      let externalBranches, internalBranches;
+      if (isExperimentOn(win, 'a4aFastFetchDoubleclickLaunched')) {
```

Comment:
Since you just set the experiment state, should you need this conditional at all?  It feels like you should just be able to do something like,

```javascript
expect(googleAdsIsA4AEnabled(win, element, 'expDoubleclickA4A',
        DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH,
        DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH))
  .to.be.false;
```